### DISCUSSION TITLE
Open links in new tab

### DISCUSSION THREAD

**Discussion by Ferdyverse** [tteck/Proxmox#1013] [Ideas]
Hey,
I think it would be a nice addition to add a target to your donation and github links so they open in a new browser tab.
`<a href="" target="_blank">LINK</a>` 

Greetings

Brazier85

**Comment by tteck** [comment]
I've never been able to get a click to open a new browser tab (from the Proxmox UI), unless I'm doing it incorrectly. I always have to use Ctrl+Click.
